CMakeLists.txt

```diff
@@ -173,17 +168,27 @@ endforeach()
 add_library(clblast SHARED ${SOURCES})
 target_link_libraries(clblast ${OPENCL_LIBRARIES})
 
+# Includes directories: CLBlast and OpenCL
+target_include_directories(clblast PUBLIC
+                           $<BUILD_INTERFACE:${clblast_SOURCE_DIR}/include>
+                           $<BUILD_INTERFACE:${clblast_SOURCE_DIR}/src>
+                           $<INSTALL_INTERFACE:include>
+                           ${OPENCL_INCLUDE_DIRS})
+
```

Comment:
This is still CMake 2.8.11 if I am correct, so it would be good to have this in anyway. But then something like this perhaps? Or am I missing something?
```
target_include_directories(clblast
                           PRIVATE
                           $<BUILD_INTERFACE:${clblast_SOURCE_DIR}/src>
                           PUBLIC
                           $<BUILD_INTERFACE:${clblast_SOURCE_DIR}/include>
                           $<INSTALL_INTERFACE:include>
                           ${OPENCL_INCLUDE_DIRS})
```

Also, then I think it is best to also use `target_include_directories` for the remainder of the targets (tests, samples, clients, tuners) further down in the CMake file.